src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -88,8 +88,10 @@ public ProCombatMoveAI(final ProAI ai) {
       clearedTerritories.add(patd.getTerritory());
     }
     territoryManager.populateEnemyAttackOptions(clearedTerritories, new ArrayList<>());
+    Set<Territory> territoriesToCheck = new HashSet<>(clearedTerritories);
+    territoriesToCheck.addAll(ProData.myUnitTerritories);
     Map<Territory, Double> territoryValueMap =
-        ProTerritoryValueUtils.findTerritoryValues(player, new ArrayList<>(), clearedTerritories);
+        ProTerritoryValueUtils.findTerritoryValues(player, new ArrayList<>(), clearedTerritories, territoriesToCheck);
```

Comment:
Yeah for now I think right now it would cause issues. I'm hoping to eventually remove the old method that doesn't take a list of territories to check in and then potentially create methods to avoid passing in empty lists.